includes/managers/controls.php

```diff
@@ -913,4 +913,50 @@ public function add_custom_css_controls( Controls_Stack $controls_stack ) {
 
 		$controls_stack->end_controls_section();
 	}
+
+	/**
+	 * Add custom attributes controls.
+	 *
+	 * This method adds a new control for the "Custom Attributes" feature. The free
+	 * version of elementor uses this method to display an upgrade message to
+	 * Elementor Pro.
+	 *
+	 * @since 2.8.3
+	 * @access public
+	 *
+	 * @param Controls_Stack $controls_stack.
+	 */
+	public function add_custom_attributes_controls( Controls_Stack $controls_stack ) {
+		$controls_stack->start_controls_section(
+			'section_custom_attributes_pro',
+			[
+				'label' => __( 'Attributes', 'elementor' ),
+				'tab' => self::TAB_ADVANCED,
+			]
+		);
+
+		$controls_stack->add_control(
+			'custom_attributes_pro',
+			[
+				'type' => self::RAW_HTML,
+				'raw' => '<div class="elementor-nerd-box">' .
```

Comment:
Make a general method that retrieves the template and use it for both custom-css and custom attributes